### DISCUSSION TITLE
Can I use home-assistant-glow with this electricity meter: MT175 ?

### DISCUSSION THREAD

**Discussion by DirkStorck** [klaasnicolaas/home-assistant-glow#186] [Q&A]
Hi,

I would like to try to read the values of my electricity meter [MT175 ](https://lackmann.de/hardware/elektrizitaetszaehler/MT175) (sorry, that it is in German)

![grafik](https://user-images.githubusercontent.com/2306190/181493693-0d18d16f-e3b0-4146-9c64-baa7f9bbb228.png)

Can I use this project for that kind of meter? It has n 

And can I use a power-bank or something like that because I live in a rental apartment with no energy source nearby? 

And the last question, can I by this somewhere build all together because I don't have the experience for that to build by my own.

Thank you!

**Comment by klaasnicolaas** [comment]
No problem it's German, currently learning the German language (vacation) 😄 

![181493693-0d18d16f-e3b0-4146-9c64-baa7f9bbb228](https://user-images.githubusercontent.com/20448157/181505498-9191725f-94f2-4cee-be8b-74975c2b76ad.png)

You can use the Home Assistant Glow on the LED I marked in blue. If I understand correctly you have a pulse rate of 10.000 imp/kWh. You could use a power bank, but I don't know how long you can use it without charging. Furthermore, I do not sell ready-made glows (although those plans are still there), but it is quite easy to assemble.

**Comment by DirkStorck** [comment]
Hi, thank you for your answer! Do you know how much energy the device needs?

**Reply by klaasnicolaas** [reply]
That is not known to me, although it probably won't be that much so if you have a large power bank you should be able to go a few weeks ahead.

**Reply by znac049** [reply]
I don't know the answer right now but am busy doing the same thing myself. I'm hoping I'll be able to run it for a few weeks at a time from a power bank, so I don't have to run power to the meter box (outside the house). I'm off on holiday for a couple of weeks but will measure the current consumed by the home-assistant-glow and publish the results here when I get back. One thing on my list of things to check is how much/many of the ESP-32's power saving options the esphome code makes use of.
